internal/bot/bot.go

```diff
@@ -57,58 +45,55 @@ func main() {
 
 	bot, err := tg.NewBotAPI(cfg.BotToken)
 	if err != nil {
-		log.Error("Failed connect to Telegram.")
-		log.Error(err)
+		log.Errorf("Failed connect to Telegram. %v", err)
 		os.Exit(3)
 	}
+	log.Infof("Authorized on telegram account @%s", bot.Self.UserName)
+
 	bot.Debug = cfg.TelegramDebug
 
-	app := App{
+	return &t.Bot{
 		Config: cfg,
 		DB:     db,
-		Bot:    bot,
+		API:    bot,
+		Log:    log,
 	}
+}
 
-	log.Infof("Authorized on telegram account @%s", bot.Self.UserName)
-
+// Start starts polling for messages for bot
+func Start(bot *t.Bot) {
```

Comment:
Maybe implement method `Start` in the `*t.Bot` like:
`func (b *Bot) Start() (err error) { return }`
And using in the main looks like 
```
//...
bot := NewBot(config)
err = bot.Start()
//...
```